binding/react-native/rhino-react-native.podspec

```diff
@@ -14,8 +14,8 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/Picovoice/rhino.git", :tag => "#{s.version}" }
 
   s.source_files = "ios/*.{h,m,mm,swift}"
-  s.vendored_frameworks = "ios/PvRhino.xcframework"
   s.resources = 'ios/resources/**/*.*'
```

Comment:
I don't think we need resources anymore, since those are coming as part of Rhino-iOS